src/workspace/src/utils.ts

```diff
@@ -11,7 +12,7 @@ export const getSpotTileApps = async (): Promise<App[]> => {
     manifestType: "url",
     manifest: `${VITE_RT_URL}/fx-spot/${symbol}`,
```

Comment:
this is the correct format for RT FX section URLs - sorry, this is my fault for not doing a global replace earlier 😞

Reply:
sorry, should have been more explicit about this .. I meant the "fx-" bit

Reply:
as opposed to the RT_URL/credit-* stuff